### PR TITLE
Add upgrade instructions for Hopsworks 1.0.0

### PR DESCRIPTION
**Pull request by SirOibaf**

### CODE REVIEW

File: docs/installation_guide/upgrades/1_0_0.rst [hunk 1]
```diff
@@ -21,6 +21,27 @@ Changes to the cluster definition
    - `hopsmonitor::node_exporter` should be added to all the nodes of the cluster
    - `hopsmonitor::purge_telegraf` should be added to all the nodes of the cluster to remove previously installed versions of telegraf.
 
+Moreover, if you add the `hopsmonitor::alertmanager` recipe, you should also add the attributes to instruct the alertmanager on to where to send the alerts. The recipes currently support email and/or slack configuration:
```
Comment (tkakantousis) [inline docs/installation_guide/upgrades/1_0_0.rst]: @SirOibaf `on to` is redundant

File: docs/installation_guide/upgrades/1_0_0.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    attrs:                                                                                                         
+        alertmanager:    
+            slack:    
+                api_url: <api_url> 
+                channel: hopsworks 
+                username: alertmanager  
+
+
+3. Hopsworks 1.0.0 supports Flink and Beam. During the upgrade you should add the `flink::yarn` and `hopslog::_filebeat-beam` recipes on all the nodes where there is a node manager (`hops::nm`) and on the Hopsworks node. `flink::historyserver` should also be added to the Hopsworks node
```
Comment (tkakantousis) [inline docs/installation_guide/upgrades/1_0_0.rst]: @SirOibaf `hopslog::_filebeat-beam` should also be on the Hopsworks machine for the community edition

File: docs/installation_guide/upgrades/1_0_0.rst [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+4. In Hopsworks 1.0.0, to speed up Spark applications startup, we cache more artifacts on the node managers. You should add `livy::install` and `hadoop_spark::certs` on all the nodes where there is a node manager (`hops::nm`). `hadoop_spark::certs` should also be added to node where Hopsworks is located.
+
+5. From Hopsworks 1.0.0 the attribute `kagent/network/interface` should be removed.
```
Comment (tkakantousis) [inline docs/installation_guide/upgrades/1_0_0.rst]: @SirOibaf `should be removed` -> `is dropped`

File: docs/installation_guide/upgrades/1_0_0.rst (hunk 1 above)
Comment (SirOibaf) [reply docs/installation_guide/upgrades/1_0_0.rst]: done

File: docs/installation_guide/upgrades/1_0_0.rst (hunk 2 above)
Comment (SirOibaf) [reply docs/installation_guide/upgrades/1_0_0.rst]: mmm, that's what's written :)

File: docs/installation_guide/upgrades/1_0_0.rst (hunk 3 above)
Comment (SirOibaf) [reply docs/installation_guide/upgrades/1_0_0.rst]: done

**Pull request merged**